attention_and_transformers.ipynb

```diff
@@ -0,0 +1,4395 @@
+{
```

Comment:
<p>nit: "For a more extensive range of attention mechanisms, you can visit this <a href="https://lilianweng.github.io/posts/2018-06-24-attention/#summary" target="_blank">link</a>." -&gt; "Dot product is only one of the ways to implement the scoring function for attention mechanisms, there is a more extensive list in <a href="https://lilianweng.github.io/posts/2018-06-24-attention/#summary" target="_blank">this</a> Blog post by Doctor Lilian Weng. For example another popular implementation is a normalized variant of the dot product where the score is divided by the square root of the number of dimensions in the hidden state."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>